pytest_django/fixtures.py

```diff
@@ -124,6 +125,7 @@ def _django_db_fixture_helper(transactional, request, django_db_blocker):
         from django.test import TestCase as django_case
 
     test_case = django_case(methodName='__init__')
+    test_case.multi_db = multi_db
```

Comment:
What difference does it make if it's set on the instance or the class?
(It's genuine curiosity, I made the change :>)